azure-pipelines-unix.yml

```diff
@@ -0,0 +1,105 @@
+--- 
+    name: $(Rev:r)
+    stages: 
+        -
+            stage: buildStage
+            displayName: 'Build Stage'
+            pool:
+                vmImage: "vs2017-win2016"
+                name: Azure Pipelines
+                demands:
+                    - msbuild
+                    - visualstudio
+                    - Cmd
+                    - vstest
+            jobs:
+                - 
+                    job: "buildJob"
+                    displayName: "Build Job"
+                    steps:
+                        - task: DownloadSecureFile@1
+                          inputs:
+                            secureFile: 'data-on-demand.snk'
+                          name: STRONGNAME_KEY
+                          displayName: "Download secure file"
+                        - task: ArtifactoryGenericDownload@3
+                          inputs:
+                            connection: 'jFrog-Artifactory'
+                            specSource: 'taskConfiguration'
+                            fileSpec: |
+                              {
+                                "files": [
+                                  {
+                                    "pattern": "public/com/squaredup/vsae-msi/VisualStudioAuthoringConsole_x86.msi",
+                                    "target": "VSAE/VisualStudioAuthoringConsole_x86.msi"
+                                  }
+                                ]
+                              }
+                        - task: PowerShell@2
+                          name: "prebuildTask"
+                          inputs:
+                            targetType: 'inline'
+                            script: |
+                              get-childItem D:\a\1\s\VSAE\com\squaredup\vsae-msi -recurse -force
+                              $date=$(Get-Date -Format "yyyyMMdd.HHmmss");
+                              Write-Host "##vso[task.setvariable variable=Build_Date;isOutput=true]$date"
+
+                              $file = "D:\a\1\s\VSAE\com\squaredup\vsae-msi\VisualStudioAuthoringConsole_x86.msi"
```

Comment:
It will be useful to use `$(SYSTEMWORKINGDIRECTORY)` instead of fix paths

Reply:
updated to use pre-defined variables instead of hardcoded paths